test/src/nyc-bin.js

```diff
@@ -689,4 +689,39 @@ describe('the nyc cli', function () {
       done()
     })
   })
+
+  it('allows alternative high and low watermarks to be configured', function (done) {
+    var args = [
+      bin,
+      '--watermarks.lines=90',
+      '--watermarks.lines=100',
+      '--watermarks.statements=30',
+      '--watermarks.statements=40',
```

Comment:
I think I can follow the logic here of setting the "low" watermark as the first arg and high as the second, but I'd prefer the DX to be more explicit, e.g.

```
--watermarks.lines.low=90
--watermarks.lines.high=100
```